google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -94,8 +94,8 @@ trait GoogleUtilities extends LazyLogging with Retry {
   }
 
   private def logGoogleRequest[A](request: AbstractGoogleClientRequest[A], startTime: Long, statusCode: Option[Int], errorReport: Option[ErrorReport]): Unit = {
-    import spray.json._
     import GoogleRequestJsonSupport._
+    import spray.json._
```

Comment:
since GoogleRequestJsonSupport is a protected member, it can't go at the top so I think I will let this ride... in this particular case it is only in the function because it is only used here